lib/fastlane/actions/mailgun.rb

```diff
@@ -26,13 +22,10 @@ def self.description
 
       def self.available_options
         [
-          FastlaneCore::ConfigItem.new(key: :mailgun_sandbox_domain,
```

Comment:
Agreed .. I'll fix that ..

On Thu, Jun 18, 2015 at 1:33 AM Josh Holtz <notifications@github.com> wrote:

> In lib/fastlane/actions/mailgun.rb
> <https://github.com/KrauseFx/fastlane/pull/355#discussion_r32698489>:
>
> > @@ -26,13 +22,10 @@ def self.description
> >
> >        def self.available_options
> >          [
> > -          FastlaneCore::ConfigItem.new(key: :mailgun_sandbox_domain,
>
> To add on to this... It might be a good idea to look for this old options
> in the action and warn the user they need to upgrade. The action suddenly
> failing might be a bit scary so a user may appreciate a nice littler
> warning/error/info/debug message [image: :whale:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/KrauseFx/fastlane/pull/355/files#r32698489>.
>